BasicSample/Assets/Rendering/Rendering.unity

```diff
@@ -1626,7 +1626,7 @@ PrefabInstance:
     - target: {fileID: 1138562410477700885, guid: 8d9a1a91bd8236b49be2be0a78fcf5e6, type: 3}
       propertyPath: Events.Array.data[2].Event.m_PersistentCalls.m_Calls.Array.data[0].m_Target
       value: 
-      objectReference: {fileID: 0}
```

Comment:
what does this change in the scene exactly?

Reply:
I was curious too - looks like when the scene remoting sample was moved to the shared package a few days ago, one of the object references / property values in the Unity scene was set to 0 (eg. the GameObjects were changed and a reference wasn't re-connected).
![image](https://user-images.githubusercontent.com/12822425/204610472-bf30d7cc-950c-41c1-9bd2-b7f03f10a030.png)

This PR looks like it's setting that value on the script again. If I had to guess from the [GameObject properties](https://github.com/microsoft/OpenXR-Unity-MixedReality-Samples/pull/125/files#diff-5386220449fcc03cf623b6b5e5002f8ad1a588b387cbd80f79d5db4bd73266d7R1562-R1564), it's maybe part of an MRTK button event?

Reply:
Yeah, it assigns the new cubes grid to the mrtk toggle receivers. The old hardcoded grid was replaced by a generated grid.